util/deprecated/v1beta1/conditions/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Should we explicitly deprecate alle exported types?

(cc @fabriziopandini)

Reply:
50, but only two different kinds:

```
import "sigs.k8s.io/cluster-api/api/v1beta1" imported as "clusterv1" but must be "clusterv1beta1" according to config (importas)
```

And

```
SA1019: "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/test/builder" is deprecated: This package is deprecated and is going to be removed when su
pport for v1beta1 will be dropped. Please see https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20240916-improve-status-in-CAPI-resources.md for more details. (staticcheck)
```

Will replace by more explicit exclusions, but also okay to work through them and fix them.

Reply:
I would really like to alert people to stop using this :)

Reply:
Update: it does on package import level, example:

```
pkg/services/vmoperator/vmopmachine_test.go:37:23: SA1019: "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions" is deprecated: This package is deprecated and is going to be removed when support for v1beta1
will be dropped. Please see https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20240916-improve-status-in-CAPI-resources.md for more details. (staticcheck)
        deprecatedconditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
```

Reply:
Ah thx. That's good enough